downloads/1.2.x-release-notes/1.2.58.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 1.2.58
+permalink: /downloads/1.2.x-release-notes/1.2.58/
+active: 1.2.58
+---
+
+## Overview of jBallerina 1.2.58
+
+<!-- COMMENT OUT ONLY THE APPLICABLE ONE FROM THE BELOW -->
+
+<!-- The jBallerina 1.2.58 patch release improves upon the <PREVIOUS_RELEASE_VERSION> release by addressing a few security improvements. -->
+
+<!-- The jBallerina 1.2.58 patch release improves upon the <PREVIOUS_RELEASE_VERSION> release by addressing a few security improvements. -->
+
```

Comment:
Need to add release details here. Check: https://github.com/ballerina-platform/ballerina-dev-website/blob/f51b8a4ee80a1764f73cecab355a039b619ddf1d/downloads/1.2.x-release-notes/1.2.53.md?plain=1#L10